lib/command/info.rb

```diff
@@ -27,6 +27,8 @@ class Info < Base # rubocop:disable Metrics/ClassLength
       ```
     EX
 
+    WITH_INFO_HEADER = false
```

Comment:
Nit: I don't think we need to separate it from the others with a newline. This applies to the other commands as well.